app/dashboard/templates/profiles/profile.html

```diff
@@ -134,7 +134,12 @@ <h1 class="profile-header__handle">
                     <div class="card-body">
                       <ul>
                         <li><b>{{ count_bounties_completed }}</b> {% trans "bounties completed" %}</li>
-                        <li><b>{{ sum_eth_collected|floatformat:2 }} ETH</b> {% trans "collected" %}</li>
+                        {% for row in sum_all_collected_tokens %}
```

Comment:
What do you mean by `row` here? Each item? When I see row I think of database rows

Reply:
I've changed the name